src/main/resources/init_db.sql

```diff
@@ -24,6 +24,7 @@ CREATE TABLE `users` (
                          `user_name` varchar(225) NOT NULL,
                          `login` varchar(225) NOT NULL,
                          `user_password` varchar(225) NOT NULL,
+                         `salt` varchar(255) DEFAULT NOT NULL,
```

Comment:
... default not null... ? did you copy this correctly?

Reply:
Fixed. Thx.